docs/launch_PMM.md

```diff
@@ -0,0 +1,24 @@
+# Launch a Percona Monitoring and Management (PMM) environment
+
+Test-drive PMM from within Percona Platform by generating a free Kubernetes cluster, automatically provisioned with a PMM server, where you can try QAN, DBaaS, Advisor, Alerts and all the features that PMM has to offer.
+
+PMM is an open-source solution for running and managing MySQL and MongoDB clusters on Kubernetes. It provides the observability required to understand database health while offering actionable insights to remediate database incidents or performance issues.  
+
+You can create one free cluster per day. The free clusters have three worker nodes, 4 CPUs/8GB of RAM, and expire in three hours. This makes them ideal for testing out PMM in a native Kubernetes experience.
```

Comment:
For this one 'You can create one free cluster per day' may be better to say - You can create one free Kubernetes environment per day with the following characteristics - three worker nodes, 4 CPUs/8GB of RAM, and it expires in three hours.